### PR TITLE
Add maths use-cases to doc

### PR DESCRIPTION
**Pull request by dladd**
Addresses issue #65.

### CODE REVIEW

**Comment by agarny** [comment]
Looks good to me.

File: docs/usecases.rst [hunk 1]
```diff
@@ -81,11 +81,11 @@ Use-cases for libCellML
          
    #. a model with maths and variables
    
-      a. … to be added to ...
+      a. model from *1.xi.d.1* and define invalid maths
```
Comment (jonc125) [inline docs/usecases.rst]: Shouldn't this be valid maths? Not that it really matters at this stage, but saying our first use case is invalid maths sounds bizarre :)

Comment (agarny) [inline docs/usecases.rst]: Hmm, indeed, well spotted!

**Comment by dladd** [comment]
Good point @jonc125 - updated now.

**Pull request merged**